livekit-agents/livekit/agents/voice/audio_recognition.py

```diff
@@ -450,9 +455,6 @@ async def _on_vad_event(self, ev: vad.VADEvent) -> None:
             with trace.use_span(self._ensure_user_turn_span()):
                 self._hooks.on_end_of_speech(ev)
 
-            self._speech_end_time = time.time()
```

Comment:
Any reason for this change? IIRC it was helping against misspredicted "huge" e2e latency

Reply:
it was not used anywhere